components/x-privacy-manager/src/radio-btn.scss

```diff
@@ -0,0 +1,83 @@
+@import 'o-colors/main';
+@import 'o-grid/main';
+@import 'o-normalise/main';
+@import 'o-spacing/main';
+@import 'o-typography/main';
+
+.input {
+	@include oNormaliseVisuallyHidden;
+}
+
+.control {
+	flex: 1;
+
+	& + .control {
+		margin-top: oSpacingByName(s4);
+
+		@include oGridRespondTo($from: S) {
+			margin-top: 0;
+			margin-left: oSpacingByName(s4);
+		}
+	}
+}
+
+.label {
+	transition: background-color 0.1s ease-in, color 0.1s ease-in;
+
+	display: flex;
+	align-items: center;
+
+	padding: oSpacingByName(s6) oSpacingByName(s4);
+	border: 2px solid oColorsByName('teal');
+	cursor: pointer;
+	background-color: oColorsByName('white');
+
+	.input:checked + & {
+		background-color: oColorsByName('teal');
+		color: oColorsByName('white');
+	}
+	
+	.input:focus + & {
+		outline: 2px solid #7aacfe;
+		outline: 5px auto -webkit-focus-ring-color;
```

Comment:
You could consider a comment explaining your reasoning for this.

Reply:
Solid suggestion 👍